tests/test_index.py

```diff
@@ -362,3 +364,27 @@ def test_linear_index_moltype_select():
     # select something impossible
     linear2 = linear.select(ksize=4)
     assert len(linear2) == 0
+
+
+@utils.in_tempdir
+def test_index_same_md5sum_fsstorage(c):
```

Comment:
This test is passing, but should probably be failing.

Reply:
Why should this fail?

Reply:
because it was writing two leaves to same file, now it creates a different file for each leaf.